src/core/gui/MainWindow.cpp

```diff
@@ -464,6 +464,14 @@ auto MainWindow::onKeyPressCallback(GtkWidget* widget, GdkEventKey* event, MainW
         // editing text - give that control
         return false;
     }
+    if (event->keyval == GDK_KEY_c && event->state & GDK_CONTROL_MASK) {
+        // Shortcut to get selected PDF text.
+        PdfFloatingToolbox* tool = win->getPdfToolbox();
+        if (tool->hasSelection()) {
+            tool->copyTextToClipboard();
+        }
+        return true;
```

Comment:
The event is only handled if there is a pdf text selection.
```suggestion
            return true;
        }
```